coalib/output/ConsoleInteraction.py

```diff
@@ -289,6 +295,14 @@ def print_results_formatted(log_printer,
                             section,
                             result_list,
                             *args):
+    """
+    Prints results in the user given format
```

Comment:
I think *args is there to sponge up any extra param that may get specified by a unified caler, so it needs no docs